### PR TITLE
Feature/prod 2955/add traces to hz cache

### PR DESCRIPTION
**Pull request by JoelNygren-Norce**

### CODE REVIEW

File: HzMemoryCache/Diagnostics/HzCacheDiagnostics.cs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+﻿namespace HzCache.Diagnostics
+{
+    public static class HzCacheDiagnostics
+    {
+        public const string HzCacheVersion = "0.0.9";
```
Comment (johan-bjerling) [inline HzMemoryCache/Diagnostics/HzCacheDiagnostics.cs]: Surely this will be 0.0.10 once this is merged?

**Review by johan-bjerling — APPROVED** [review]
Possibly change the version string though.

File: HzMemoryCache/Diagnostics/HzCacheDiagnostics.cs (hunk 1 above)
Comment (JoelNygren-Norce) [reply HzMemoryCache/Diagnostics/HzCacheDiagnostics.cs]: maby we should remove this as no one will keep it updated?

**Review by patrik-j-nilsson — CHANGES_REQUESTED** [review]
I think it's a bit unnecessary to have a dependency on OpenTelemetry in this package, then we need to keep it up to date here as well.

The extension method is a nice idea, but I fear it could create more problems than it's worth.
It should be as simple as adding `builder.AddSource(HzCacheDiagnostics.ActivitySourceName);` in the service that uses this package.

**Pull request merged**